packages/wrangler/src/index.tsx

```diff
@@ -1871,5 +1871,11 @@ export async function main(argv: string[]): Promise<void> {
 
   await initialiseUserConfig();
 
-  await yargs.parse(argv);
```

Comment:
could you explain this? we want errors to exit the process with signal 1, don't we?

Reply:
This was a bit of a mess, so I spent some time sorting it all out at https://github.com/cloudflare/wrangler2/pull/164/commits/a67058d5ffca5ca6a01686385e3c71e7537694d2